central/reports/scheduler/schedule.go

```diff
@@ -72,12 +73,16 @@ var (
     }`
 
 	vulnReportEmailTemplate = `
-	Red Hat Advanced Cluster Security for Kubernetes has found vulnerabilities associated with the running container images owned by your organization. Please review the attached vulnerability report {{.WhichVulns}} for {{.DateStr}}.
+	%s has found vulnerabilities associated with the running container images owned by your organization. Please review the attached vulnerability report {{.WhichVulns}} for {{.DateStr}}.
```

Comment:
The string with two types of templates: `%s` and `{{.Something}}` looks strange and questionable. Could you use only one type of templating, please?

Reply:
Good point, I overlooked the templating at the end of the string. Will change it accordingly.

Reply:
Templating should be consistent now, also improved the tests.